.cirrus.yml

```
@@ -0,0 +1,13 @@
+freebsd_task:
+  freebsd_instance:
+    matrix:
+    - image: freebsd-13-2-release-amd64
+  build_script:
+    - mkdir build
+    - cd build
+    - ../configure
+    - make -j$(sysctl -n hw.ncpu)
+  test_script:
+    - cd build
+    # dtrace fails to build on FreeBSD - see gh-73263
+    - make buildbottest TESTOPTS="-j2 -u-network -x test_dtrace"
```

You can ask Python to select the number of worker processes using the number of detected CPUs: ``-j0`` option.

```suggestion
    - make buildbottest TESTOPTS="-j0 -u-network -x test_dtrace"
```

Apparently, Cirrus CI announces 2 CPUs, so Python will spawns 4 processes in parallel. It may better use resources, since many tests "sleep" for different reasons.